### ISSUE TITLE
JIRA > WORFRONT

### ISSUE THREAD

**Issue by gjungberg** [cthellewell/WorkfrontSync#1]
Hi,

I would like to know if this is a plugin to connect jira and workfront for project sync. 

Thanks.

**Comment by cthellewell** [comment]
This code does sync Jira projects with Workfront projects, but it is very customized and tailored to fit our company's specific needs.

**Comment by gjungberg** [comment]
I understand that. This is Java right?. My other question is you need to have always a server running to do the sync.

Thanks

**Comment by cthellewell** [comment]
Yes, this is Java code. I run this as a Windows service on the same server
with our on premise instance of Jira.

Chris
